app/views/hyrax/base/_guts4form.html.erb

```diff
@@ -0,0 +1,49 @@
+<%# Hyrax v5.0.1 Override - adds switching of tab delivery if user is admin or not.
+    Hyrax dev note: we will yield to content_for for each tab, e.g. :files_tab %>
+<% parsed_tabs = current_user.admin? ? (tabs - ['share']) : (tabs - ['relationships', 'share']) %>
```

Comment:
@bwatson78 is there a simple way to do this via controller instead of directly through the view? also, have we looked into the endpoints used in the relationships and share tab, and ensured access is also limited to admin users for those endpoints? e.g. an admin can query all collections and admin sets in the relationships tab, have we tested that users cannot run the same queries?

Reply:
The QA seems to be wide open and I can't find an easy way or restricting its controller actions without bringing in the whole controller as an override.

Reply:
@bwatson78 the changes to `config/initializers/hyrax_work_form_helper_override.rb` look good to me. As far as restricting access to non-admin users go, it may be a good topic to discuss with Emily. If possible, it may be better to create a new ticket to implement those restrictions and leave this ticket focused on the changes in UI. If a new ticket is created for access control, I will approve and merge this PR.